src/averageAges.js

```diff
@@ -20,6 +20,20 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const allMen = people.filter(person => person.sex === 'm');
```

Comment:
U can use ternary here and if `century` was passed just do the correct filter condition, to prevent repetitive logic in return